### PR TITLE
Task-J66: Added podcast page

### PR DESCRIPTION
**Pull request by vitalik-prog**

### CODE REVIEW

**Review by TarasMakarchuk — DISMISSED** [review]

File: frontend/src/common/enums/app/app-route.enum.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ enum AppRoute {
   SIGN_IN = '/sign-in',
   SIGN_UP = '/sign-up',
   PODCAST = '/podcast',
-  PODCAST_$ID = '/podcasts/:id',
+  PODCAST_$ID = '/podcast/:id',
```
Comment (ghost) [inline frontend/src/common/enums/app/app-route.enum.ts]:
```suggestion
  PODCASTS_$ID = '/podcasts/:id',
```
and change according to the documentation for all keys

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/components/podcast/podcast.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { useAppSelector, useDispatch, useEffect, useParams } from 'hooks/hooks';
+import { podcast as podcastActions } from 'store/actions';
+import { PageParams } from './common/types/types';
+import styles from './styles.module.scss';
+
+const Podcast: React.FC = () => {
+  const dispatch = useDispatch();
+  const { podcastId } = useParams<PageParams>();
+
+  const { podcast } = useAppSelector(({ podcast }) => ({
```
Comment (ghost) [inline frontend/src/components/podcast/podcast.tsx:10]: return there is no podcast if we don't have podcast with such a id

File: frontend/src/components/podcast/podcast.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    podcast: podcast.podcast,
+  }));
+
+  useEffect(() => {
+    dispatch(podcastActions.loadPodcast(Number(podcastId)));
+  }, []);
+
+  return (
+    <main className={styles.podcast}>
+      podcast here: {podcast?.name}
```
Comment (ghost) [inline frontend/src/components/podcast/podcast.tsx]:
```suggestion
      <h1>{podcast.name}</h1>
      <p>{podcast.description}</p>
```

File: frontend/src/common/enums/app/app-route.enum.ts [hunk 4]
```diff
@@ -2,8 +2,8 @@ enum AppRoute {
   ROOT = '/',
   SIGN_IN = '/sign-in',
   SIGN_UP = '/sign-up',
-  PODCAST = '/podcast',
-  PODCAST_$ID = '/podcasts/:id',
+  PODCASTS = '/podcasts',
+  PODCASTS_$ID = '/podcasts/:podcastId',
```
Comment (ghost) [inline frontend/src/common/enums/app/app-route.enum.ts]:
```suggestion
  PODCASTS_$ID = '/podcasts/:id',
```just it

File: frontend/src/components/podcast/common/types/page-params.type.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+type PageParams = {
+  podcastId: string;
```
Comment (ghost) [inline frontend/src/components/podcast/common/types/page-params.type.ts]:
```suggestion
  id: string;
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**